runtime-sdk/modules/evm/Cargo.toml

```diff
@@ -18,8 +18,14 @@ thiserror = "1.0"
 hex = "0.4.2"
 sha2 = "0.9.5"
 ripemd160 = { version = "0.9", default-features = false }
-k256 = { version = "0.10.4", default-features = false, features = ["keccak256", "ecdsa"] }
+k256 = { version = "0.10.4", default-features = false, features = [
+    "keccak256",
+    "ecdsa",
+] }
 sha3 = { version = "0.10", default-features = false }
+substrate-bn = "0.6.0"
+serde = { version = "1.0.144", features = ["derive"] }
+serde_json = { version = "1.0.87", features = ["raw_value"] }
```

Comment:
Are these only needed for tests? In that case they should be moved under `dev-dependencies` below.